packages/backend/src/queryBuilder.test.ts

```diff
@@ -397,9 +397,9 @@ describe('with custom dimensions', () => {
     it('getCustomDimensionSql with empty custom dimension', () => {
         expect(
             getCustomDimensionSql({
+                warehouseClient: bigqueryClientMock,
```

Comment:
by default I'm testing with bigquery, which is going to be using the default CONCAT method, however, the `quotes` are different in bigquery, so I had to update most of the SQL statements.